src/server/DredReplicator.ts

```diff
@@ -713,6 +731,101 @@ export class Replicant {
         }
     }
 
+    /**
+     * Handle channel events from _chans meta channel
+     */
+    private handleChannelEvent(message: any): void {
+        try {
+            const { type, msg } = message;
+
+            if (type === 'chanCreated') {
+                const data = JSON.parse(msg);
+                const { channel, options } = data;
+
+                this.log(`📢 Channel creation detected on ${this.targetHost.serverId}: ${channel}`);
+
+                // Skip meta channels (they start with _)
+                if (channel.startsWith('_')) {
+                    this.debug(`Skipping meta channel: ${channel}`);
+                    return;
+                }
+
+                // Handle asynchronously but don't await to avoid blocking
+                this.handleChannelAdded(channel, options).catch(error => {
+                    this.warn(`Error handling channel addition: ${error}`);
+                });
+            }
+        } catch (error) {
+            this.warn(`Error handling channel event: ${error}`);
+        }
+    }
+
+    /**
+     * Handle a new channel being added on the target server
+     */
+    private async handleChannelAdded(channelName: string, options: any): Promise<void> {
+        try {
+            // Check if home server already has this channel
+            const hasChannel = await this.homeServer.channelList.has(channelName);
+
+            if (hasChannel) {
+                this.debug(`Channel ${channelName} already exists on home server`);
+                return;
+            }
+
+            this.log(`🆕 Creating channel ${channelName} on home server ${this.homeServer.serverId}`);
```

Comment:
I'd suggest using the ("... message %s", channel) way of expressing log parameters.  As for the server id:

<img width="363" height="52" alt="image" src="https://github.com/user-attachments/assets/71645a5d-4cfb-4db5-b6fa-5fb5f9e6ffab" />

I think the logger-id already indicates the server-id (I think "first' will look like `dredNode-XyZpDQsweet1` in a production-like environment).  

It can be a little hard to remember when details are already implicit in the logger, but I'll keep reminding you

Reply:
looking pretty good overall